Fakes/Fakes.xcodeproj/project.pbxproj

```diff
@@ -9,7 +9,6 @@
 /* Begin PBXBuildFile section */
 		26106B3D25FA4F6C0000DF30 /* ProductFactory.swift in Sources */ = {isa = PBXBuildFile; fileRef = 26106B3C25FA4F6C0000DF30 /* ProductFactory.swift */; };
 		263E383C2641FED600260D3B /* Codegen in Frameworks */ = {isa = PBXBuildFile; productRef = 263E383B2641FED600260D3B /* Codegen */; };
-		263E383D2641FED600260D3B /* Codegen in Embed Frameworks */ = {isa = PBXBuildFile; productRef = 263E383B2641FED600260D3B /* Codegen */; settings = {ATTRIBUTES = (CodeSignOnCopy, ); }; };
```

Comment:
These changes can only be done via Xcode 12.5 GUI, because if you try to remove Codegen from Embed Frameworks using Xcode 12.4, it will also remove it from the "Link Binary with Libraries" phase too.

That's an Xcode 12.4 bug that was fixed in Xcode 12.5, so the trick was to fix the pbxproj using Xcode 12.5, yet still continue to use Xcode 12.4 for development and submission – as luckily opening the `pbxproj` in Xcode 12.4 (after having fixed it in Xcode 12.5) didn't revert the fix nor reintroduce the bug in Xcode 12.4 😓